mailpoet/assets/css/src/components-plugin/_common.scss

```diff
@@ -122,3 +122,26 @@ body .components-modal__screen-overlay {
   justify-content: flex-end;
   margin-top: $grid-gap-half;
 }
+
+.mailpoet-locked-badge {
+  align-items: center;
+  background: #fcf9e8;
+  border: .5px solid #f5e6ab;
+  border-radius: 4px;
+  box-sizing: border-box;
+  color: #bd8600;
+  display: flex;
+  flex: none;
+  flex-direction: row;
+  flex-grow: 0;
+  font-size: 11px;
+  font-weight: 500;
+  gap: 4px;
+  height: 20px;
+  letter-spacing: .2px;
+  line-height: 16px;
+  order: 0;
+  padding: 2px 8px 2px 4px;
+  text-transform: uppercase;
+  width: 82px;
+}
```

Comment:
This may come from Figma, but I feel like some of the properties are not necessary:
- `order: 0`
- `flex-direction` (`row` is the default)
- `flex-grow: 0` (covered by `flex: none`)
- `box-sizing: border-box` (I hope our default)

Reply:
[c8ed429](https://github.com/mailpoet/mailpoet/pull/4604/commits/c8ed42915be64132cb3cd913efa1e859d6c217fa)